### ISSUE TITLE
Creation de lot - Parfois pas de dialog box (est-ce voulu?)

### ISSUE THREAD

**Issue by jocelyncardinal** [CAUCA-9-1-1/survip#359]
1. Lors de la création d'un lot, le système n'affiche plus le dialog box et on redirigé plutôt a la liste de lot. Note que parfois on affiche encore le dialog - C'est pas clair, le comportement est inconsistant 
2. lors de la redirection a la page de gestion de lot, il n'y a pas de titre a la page et pas de menu correspondant ...

![Capture d’écran 2018-10-26 à 11.25.14.png](https://images.zenhubusercontent.com/5aeb25264b5806bc2bd1a294/35cfaf1f-ad59-4c74-a6bd-8621cc2f0186)

![Capture d’écran 2018-10-26 à 11.43.14.png](https://images.zenhubusercontent.com/5aeb25264b5806bc2bd1a294/dc7fa3d0-2b00-41c9-a27a-bb691a86f16e)

**Comment by iamkinetic** [comment]
@philipperobertgh Tu m'as dit avoir observé ce comportement toi aussi, n'est-ce pas?  As-tu vu des erreurs quelconques?

**Comment by iamkinetic** [comment]
@jocelyncardinal Est-ce que tu as un lot qui te donne ce résultat comme c'est là?

**Comment by jocelyncardinal** [comment]
Non, je me rappelle pas avoir vu une erreur ... ce comportement par tout le temps ...  attend un peu je vais essayer de le reproduire ...

**Comment by jocelyncardinal** [comment]
comportement reproduit avec lot test issue#359a (usager/ password = patrick.giguere/ereugig1234)

**Comment by jocelyncardinal** [comment]
Comportement très bizarre ensuite ... Toutes les colonnes deviennent sélectionnées ... voir issue #376

**Comment by iamkinetic** [comment]
Je ne vois pas ces lots en test.  Est-ce qu'ils ont été supprimés?

**Comment by jocelyncardinal** [comment]
Non, ils n'ont pas été supprimé ... essais avec l'usager patrick giguere (ereugig1234)

![capture d ecran 2018-11-05 a 08 31 09](https://user-images.githubusercontent.com/32175949/48001042-67dd1c80-e0d5-11e8-873a-b82a5e4ba9d6.png)

![capture d ecran 2018-11-05 a 08 31 46](https://user-images.githubusercontent.com/32175949/48001050-6c093a00-e0d5-11e8-9043-40136b3c9f33.png)

**Comment by iamkinetic** [comment]
Ok, c'est parce que les cas sont dans Waterville et que mon usager n'avait pas accès à ce SSI.

Je peux voir que les lots problématiques causent un message d'erreur dans la console du browser lorsqu'on essai de les ouvrir.  L'erreur est aussi vague que possible, mais on va regarder ça plus en détail.

```
vendor.js:56112 ERROR TypeError: Cannot read property 'option' of undefined
    at main.js:4080
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (polyfills.js:2743)
    at Object.onInvokeTask (vendor.js:58254)
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (polyfills.js:2742)
    at Zone.push../node_modules/zone.js/dist/zone.js.Zone.runTask (polyfills.js:2510)
    at push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask (polyfills.js:2818)
    at ZoneTask.invoke (polyfills.js:2807)
    at timer (polyfills.js:4376)
```

**Comment by iamkinetic** [comment]
Ahh, on vient de comprendre!  Dans la plupart des tableaux de bords, on utilise le paging pour éviter de loader trop de données à la fois.  Tous les lots qui ne s'ouvrent pas ne sont pas dans la première  page loadée du tableau de bord de lots.

Nous allons devoir modifier la façon d'ouvrir le lot pour s'assurer que le lot qu'on veut ouvrir soit toujours loadé quand on ouvre le tableau de bord des lots.

**Comment by jocelyncardinal** [comment]
Oui, logique car mes lots à problème étaient tous dans la 2ème page.
